Source/VisualRecognitionV4/Models/ImageDetails.swift

```diff
@@ -24,17 +24,17 @@ public struct ImageDetails: Codable, Equatable {
     /**
      The identifier of the image.
      */
-    public var imageID: String
+    public var imageID: String?
```

Comment:
This is also technically a breaking change, along with the properties below have been changed to optionals.  Are we okay with this?

Reply:
Looks like these were intentionally marked as optional in the API definition. 

```
https://github.ibm.com/Watson/developer-cloud--api-definitions/pull/1488
```

Depending on the operation these fields in the response may not be returned.